coalib/processes/BearRunning.py

```diff
@@ -5,8 +5,8 @@
 from coalib.bears.GlobalBear import GlobalBear
 from coalib.bears.LocalBear import LocalBear
 from coalib.misc import Constants
-from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
 from coalib.processes.CONTROL_ELEMENT import CONTROL_ELEMENT
+from coalib.processes.communication.LogMessage import LOG_LEVEL, LogMessage
```

Comment:
if the thread is removed we don't need the message right? Suggestion: do the removal of that in a seperate PR, this one's big enough